webpack/assets/javascripts/react_app/redux/reducers/index.js

```diff
@@ -8,6 +8,7 @@ import toasts from './toasts';
 import { reducers as passwordStrengthReducers } from '../../components/PasswordStrength';
 import { reducers as breadcrumbBarReducers } from '../../components/BreadcrumbBar';
 import { reducers as searchBarReducers } from '../../components/SearchBar';
+import { reducers as dualListReducers } from '../../components/DualList';
```

Comment:
Am I missing something ? I dont see any reducers in `index.js`

Reply:
they will be added later, still a very very early WIP.

Reply:
didn't see WIP 💃